cmd/handlers.go

```diff
@@ -146,6 +146,7 @@ func initHTTPHandlers(e *echo.Echo, app *App) {
 	g.PUT("/api/campaigns/:id/status", handleUpdateCampaignStatus)
 	g.PUT("/api/campaigns/:id/archive", handleUpdateCampaignArchive)
 	g.DELETE("/api/campaigns/:id", handleDeleteCampaign)
+	g.GET("/api/campaigns-report", handleGetCampaignsReport)
```

Comment:
/api/campaigns-report

change to 

/api/campaigns/report

Reply:
Done